display/pfobjects.py

```diff
@@ -116,8 +123,12 @@ def draw(self, projection, opt=''):
             
 class GStraightTrajectory(GTrajectory):
     def __init__(self, description):
-        super(GStraightTrajectory, self).__init__(description,
-                                                  linestyle=2, linecolor=1)
+        self.desc = description
+        if abs(self.desc.pdgid()) in [22]:
+          super(GStraightTrajectory, self).__init__(description,
+                                                  linestyle=2, linecolor=2)
+        elif abs(self.desc.pdgid()) in [12, 14, 16]:
```

Comment:
I thought that we used simulated particles in the display that's why I added the neutrinos. Indeed, as we used the reconstructed particles, display the neutrinos is useless.

However, I can see on the displays some dot black lines. These lines reconstructed which kind of particles?